src/test/kotlin/uk/co/craigbass/pratura/acceptance/BasketSpec.kt

```diff
@@ -22,57 +22,60 @@ class BasketSpec : Spek({
     )
   }
 
-  given("no lineItems are added to the basket") {
-    it("should contain no lineItems") {
-      basketContents().lineItems.shouldBeEmpty()
-    }
-
-    it("should have a total value of £0.00") {
-      basketContents().basketValue.shouldEqual("£0.00")
-    }
-  }
-
-  given("one item is in the basket") {
+  given("store currency is EUR") {
     beforeEachTest {
       pratura().executeUseCase(
-        AddItemToBasket::class,
-        AddItemToBasket.Request(
-          quantity = 1,
-          sku = "productsku"
+        SetStoreCurrency::class,
+        SetStoreCurrency.Request(
```

Comment:
I'm not sure how I feel about the currency of the shop being state held within Pratura itself,
it feels like something you'd store in an application specific manner and pass into the core of the system each time